### PR TITLE
Add tests for stata and fix a few bugs

### PR DESCRIPTION
**Pull request by bolliger32**
@sannanphan @jeanettelt I wanted to be able to pass the number of bootstraps to use for the r2 analysis (if any) as a command line variable, because this made it easier to include in the `run.sh` script. I did this (see files changed in this PR) but in doing so noticed that the `FRA_adm1.do` file is a bit different than the others and is not outputting the boottstrap sample results. As a result, it's empty in `ExtendedDataFigure5_lags.xlsx (panel_b)`. I couldn't quite follow the stata and didn't want to screw things up in case france was intentionally different, but could one of you pull this branch (`update/ed5-bs`) and see if you can get it to output to `results/source_data/indiv` in the same way as the other files are now doing? And then just push back to this PR.

### CODE REVIEW

**Comment by bolliger32** [comment]
OK the r2 lags issue has been fixed (Thanks @sannanphan !)

In this PR I've also switched to using a CI runner on shackleton, so that we can include the stata code in the tests. It now should be running the stata code as well, so we should have a full start to finish testing pipeline.

File: code/models/run_all_CB_simulations.R [hunk 1]
```diff
@@ -2,7 +2,7 @@
 args = commandArgs(trailingOnly=TRUE)
 
 if(length(args) > 0){
-  times <- args
+  times <- strtoi(args)
```
Comment (kendonB) [inline code/models/run_all_CB_simulations.R:5]: Approve. note that this is robust for multiple different types for `args`.

File: code/models/run_projection_with_multiple_gammas.R [hunk 2]
```diff
@@ -1,6 +1,9 @@
 suppressPackageStartupMessages(library(tidyverse))
 suppressPackageStartupMessages(library(ggsci))
 suppressPackageStartupMessages(library(patchwork))
+
+dir.create("results/figures/fig4", recursive=TRUE, showWarnings=FALSE)
```
Comment (kendonB) [inline code/models/run_projection_with_multiple_gammas.R:5]: Approve.

**Comment by kendonB** [comment]
my scripts look good.  I am also happy to hit the final button once everyone else has done their reviews of their bits.

**Comment by jeanettelt** [comment]
Do we need to change the Stata code to export .jpg then?

**Comment by trinetta** [comment]
I think the current capture command to suppress ```translator Graph2png not found``` error allows the tests to continue running (someone please correct me if I'm wrong!) This is stata's [docmentation](https://www.stata.com/manuals13/g-2graphexport.pdf)
```
ps and eps are available for all versions of Stata; png and tif are available for all versions
of Stata except Stata(console) for Unix; pdf is available only for Stata for Windows and
Stata for Mac; and wmf and emf are available only for Stata for Windows.
```
Not sure about their treatment of .jpg files, but png seems to be the most flexible version already (since not everyone wants .ps or .eps?)

**Comment by bolliger32** [comment]
thanks for taking a look @trinetta ! Yeah @jeanettelt we're already exporting them in pdf so I think we're fine. If someone really wants the png they can just run it in non-console mode.

**Comment by bolliger32** [comment]
Removing stata tests for security

**Pull request merged**